.github/workflows/pr-merge.yml

```diff
@@ -33,5 +33,5 @@ jobs:
           bz_product: "Migration Toolkit for Virtualization"
           title: ${{ github.event.pull_request.title }}
           github_token: ${{ secrets.GITHUB_TOKEN }}
-          branch_to_release: "release-v2.0.0-beta.0:2.0.0-beta.1,main:2.0.0"
+          branch_to_release: "release-v2.0.0:2.0.0,release-v2.1.0:2.1.0,main:2.1.0"
```

Comment:
```suggestion
          branch_to_release: "release-v2.0.0:2.0.0,release-v2.1.0:2.1.0,main:2.2.0"
```
Is `main:2.1.0` correct or should it be `:2.2.0`?

Reply:
@rayfordj We don't have a 2.2.0 release yet (AFAIK), that's why I left the pointer to latest known release. Is this something that can cause issues ? Can I set an nonexistent tentatively future release on this one?

Reply:
@fbladilo , AFAIK, it should be safe to have multiple GH repo branches map to the same BZ target release.  We do similar with Crane/MTC for our default branches (`master` & `konveyor-dev`).  

![image](https://user-images.githubusercontent.com/4580787/127232712-cf1f4f55-e7ad-4077-a5be-d58cdd1a3f13.png)
I ask because it appears a BZ `2.2.0` target release exists.  I wasn't sure how you expected it to work, re: "things on main", so that's fine if it was intentional (which is why I still gave the 'ack').  :sunglasses:

Reply:
@rayfordj Thanks for clarifying there, good to know target branches seem pre-created. I think I'm going to keep the 2.1.0 for now in the main branch mapping,I would like to have that conversation with the team in how to handle main once the release branches are cut. Right now we have plenty of work coming in on main branch and getting picked for release-v2.1.0.